source/documentation/tools/jupyterlab/package-management.md

```diff
@@ -1,5 +1,89 @@
 # Python package management
 
+## Poetry
+We advise the use of [poetry](https://python-poetry.org/docs/) as a library.
+
+### <b>why use poetry</b> 
+poetry stops conflicts between libraries and their dependencies.
+It makes sure that libraries play well together.
+If virtualenv is used without poetry, you may start experiencing library version conflict and start getting warnings.
```

Comment:
I'm not sure virtualenv is explained until a later section. I wonder if this might best be placed as an alternative (rather than advisory) after the Basic usage at the bottom?